docs/reference/release-notes/7.0.0-rc1.asciidoc

```diff
@@ -1,6 +1,203 @@
 [[release-notes-7.0.0-rc1]]
 == {es} version 7.0.0-rc1
 
-Also see <<breaking-changes-7.0,Breaking changes in 7.0>>.
+coming[7.0.0-rc1]
+
+Also see <<breaking-changes-7.0.0-rc1,Breaking changes in 7.0.0-rc1>>.
+
+[[breaking-7.0.0-rc1]]
+[float]
+=== Breaking changes
+
+Distributed::
+* Remove cluster state size {pull}40061[#40061] (issues: {issue}39806[#39806], {issue}39827[#39827], {issue}39951[#39951], {issue}40016[#40016])
+* Stop returning cluster state size by default {pull}40016[#40016] (issues: {issue}39806[#39806], {issue}39827[#39827], {issue}39951[#39951])
```

Comment:
I'd remove this one since it's essentially the 6.7 bw layer which is superseded by #40061.

Reply:
Thanks, removed in https://github.com/elastic/elasticsearch/pull/40479/commits/a3874174eae54a4d7e2ba40de28f46dd36848f99